src/quo/components/settings/settings_item/style.cljs

```diff
@@ -30,9 +30,10 @@
    :justify-content   :flex-start})
 
 (defn image-container
-  [image tag description]
-  {:height     (if (= image :icon-avatar) 32 20)
-   :margin-top (if (or tag description) 1 0)})
+  [image image-right? tag description]
+  {:height      (if (= image :icon-avatar) 32 20)
+   :margin-top  (if (or tag description) 1 0)
+   :margin-left (if image-right? 8 0)})
 
```

Comment:
`if` conditions in styling with zeroes can be simplified to:
```
{:height      (if (= image :icon-avatar) 32 20)
   :margin-top  (when (or tag description) 1)
   :margin-left (when image-right? 8)}
```

Reply:
Oh nice, good suggestion, thank you 😸